lib/engine/round/g_1846/draft.rb

```diff
@@ -1,125 +1,22 @@
 # frozen_string_literal: true
 
-require_relative '../base'
+require_relative '../auction'
 
 module Engine
   module Round
     module G1846
-      class Draft < Base
-        attr_reader :companies, :index
-
-        def initialize(entities, game:, **_kwargs)
+      class Draft < Auction
+        def initialize(game, steps)
```

Comment:
can you remove the initialize call and just override def select to reorder the players in reverse order?

any changes to state can be done in setup or round_state

Reply:
ironically that's how i originally coded it, then there was problems of player order after exiting the draft. I'll have another stab at it.